base/client.jl

```diff
@@ -299,9 +302,7 @@ function exec_options(opts)
     end
     repl |= is_interactive::Bool
```

Comment:
`repl` is only a local variable and seems not to have any effect before because it is always used as `repl || is_interactive::Bool`. So I moved it to the following `if `statement to simplify the code slightly.